final/implementation/guice.tex

```diff
@@ -0,0 +1,26 @@
+\section{Dependency Injection}
+\label{sec:injection}
+
+The design of Spoofax relies quite heavily on dependency injection by the Guice
+framework.  To achieve as much interopability as possible, the final product
+tries to use dependency injection in the same spirit as Spoofax. Guice has had a
```

Comment:
"Tries to" sounds as if this hasn't been achieved. I think we managed to use dependency injection quite good so I'd use "uses" instead.